### PR TITLE
Add theme switcher to redux

### PR DESCRIPTION
**Pull request by JaneEllison**
Refactor switching themes. Add themes to redux.

### CODE REVIEW

**Review by anton-vlasik — APPROVED** [review]

File: src/layouts/Header/ThemeSwitcher/Container.jsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+const ThemeSwitcherContainer = ({ theme, switchTheme }) => {
+  function changeTheme() {
+    const currentTheme = theme === THEMES.light ? THEMES.dark : THEMES.light;
+
+    switchTheme(currentTheme);
+  }
+
+  useEffect(() => {
+    localStorage.setItem('theme', theme);
```
Comment (ximet) [inline src/layouts/Header/ThemeSwitcher/Container.jsx]: How it works?

File: src/redux/reducers/themeReducer.js [hunk 2]
```diff
@@ -1,16 +1,18 @@
-import { THEMES } from '../../constants/themes';
 import { SET_CURRENT_THEME } from '../types/themeTypes';
 
+import { THEMES } from '../../constants/themes';
+import { localStorageService } from '../../services/localStorageService';
+
 const INITIAL_STATE = {
-  theme: THEMES.light
+  currentTheme: localStorageService.getCurrentTheme() || THEMES.light
```
Comment (ximet) [inline src/redux/reducers/themeReducer.js:7]: initial state from localStorage? Why?

File: src/layouts/Header/ThemeSwitcher/Container.jsx (hunk 1 above)
Comment (JaneEllison) [reply src/layouts/Header/ThemeSwitcher/Container.jsx]: If you change theme, theme name save to local storage, and then if you refresh page you see this one which you choose earlier

File: src/redux/reducers/themeReducer.js (hunk 2 above)
Comment (JaneEllison) [reply src/redux/reducers/themeReducer.js:7]: If the user refresh the page, state will set saved theme value. 
Or I should do this in the component?

const currentTheme=  localStorageService.getCurrentTheme();

 useEffect(() => {
    switchTheme(currentTheme)
  }, []);

Something like this?

File: src/layouts/Header/ThemeSwitcher/Container.jsx (hunk 1 above)
Comment (JaneEllison) [reply src/layouts/Header/ThemeSwitcher/Container.jsx]: I moved setting data to localStorageService, removed useEffect and calling the function on changeTheme function.

**Review by ximet — APPROVED** [review]

**Pull request merged**